tools/manta.cwl

```diff
@@ -0,0 +1,96 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: manta 
+doc: "Calls structural variants."
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: $(inputs.ram * 1000)
+    coresMin: $(inputs.cpu)
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/manta:1.6.0'
+  - class: InitialWorkDirRequirement
+    listing:
+    - entryname: rename_outs.sh
+      writable: false
+      entry: |
+        set -eu
+        set -o pipefail
+
+        mv results/variants/*candidateSV.vcf.gz $(inputs.output_basename).manta.candidateSV.vcf.gz  || :
+        mv results/variants/*candidateSV.vcf.gz.tbi $(inputs.output_basename).manta.candidateSV.vcf.gz.tbi || :
+        mv results/variants/*diploidSV.vcf.gz $(inputs.output_basename).manta.diploidSV.vcf.gz || :
+        mv results/variants/*diploidSV.vcf.gz.tbi $(inputs.output_basename).manta.diploidSV.vcf.gz.tbi || :
+        mv results/variants/*somaticSV.vcf.gz $(inputs.output_basename).manta.somaticSV.vcf.gz || :
+        mv results/variants/*somaticSV.vcf.gz.tbi $(inputs.output_basename).manta.somaticSV.vcf.gz.tbi || :
+        mv results/variants/*candidateSmallIndels.vcf.gz $(inputs.output_basename).manta.candidateSmallIndels.vcf.gz || :
+        mv results/variants/*candidateSmallIndels.vcf.gz.tbi $(inputs.output_basename).manta.candidateSmallIndels.vcf.gz.tbi || :
```

Comment:
why do you need to move and rename them ?

Reply:
Manta doesn't let you make custom names for your outputs so I'm doing that myself. We're renaming them because these are outputs that will be returned to the user and we want to give them pleasing naming.